design/operator.md

```diff
@@ -0,0 +1,65 @@
+# Observerability
+
+* **Status**: PoC
+* **Author**: [slaskawi](https://github.com/slaskawi)
+* **JIRA**: [KEYCLOAK-10036](https://issues.jboss.org/browse/KEYCLOAK-10036)
+* **PoC video**: https://www.youtube.com/watch?v=nGsARoHk_Dw
+
+## Motivation
+
+* Simplify user experience on OpenShift
```

Comment:
Is this intended to be restricted to just OpenShift? I would be interested in using it on Kubernetes as well.

Reply:
That's for Kubernetes as well. Let me clarify this a bit more here.